slack/web/async_client.py

```diff
@@ -183,10 +183,30 @@ async def admin_conversations_restrictAccess_removeGroup(
             params=kwargs,
         )
 
-    async def admin_conversations_setTeams(self, **kwargs) -> AsyncSlackResponse:
-        """Set the workspaces in an Enterprise grid org that connect to a channel."""
+    async def admin_conversations_setTeams(
+        self, *, channel_id: str, **kwargs
```

Comment:
I added `channel_id` here to match the documentation because I noticed it didn't match 
https://api.slack.com/methods/admin.conversations.setTeams